### DISCUSSION TITLE
Artillery with playwright

### DISCUSSION THREAD

**Discussion by chinchusarin** [artilleryio/artillery#1392] [Artillery]
I am using artillery to run the playwright engine. I have to test one to one twillio connection and facing issue with granting permission to microphone. Right now, I have fixed by bypassing the page created by artillery and creating a new browser context by playwright and granting permission by playwright. But this will open the chromium browser locally. I want to load test, hence want to know if there is any artillery way to grant permission for media on the page  by giving some args in the yaml file.  Hope my query is clear

**Comment by hassy** [comment]
hi @chinchusarin 👋 do you have a snippet of code that shows how you grant those permissions?

**Comment by chinchusarin** [comment]
I got it done using the xbfv command . Thankyou
